src/layout/content/content.tsx

```diff
@@ -17,11 +18,17 @@ export class MainContent extends Component<MainContentProps> {
     if (!contentElement) {
       return null
     }
+
     const topBar: HTMLElement = document.querySelector(this.props.topBarSelector)
-    const height = topBar
-      ? Math.floor(window.innerHeight - topBar.offsetHeight)
-      : window.innerHeight
+    let minus = 0
+
+    if (this.mainContentRef.current.closest('.kirk-modal--hasCloseButton')) {
+      minus = 70 // padding-top of modals with close button
```

Comment:
**No magic numbers**: is that possible to create a variable for that?

Reply:
Added the size into `_utils/banding`'s `componentSizes`